src/docs/get-started/test-drive/_vscode.md

```diff
@@ -54,4 +60,6 @@ contains a simple demo app that uses [Material Components][].
 [Quickly switching between Flutter devices]: https://dartcode.org/docs/quickly-switching-between-flutter-devices
 [status bar]: {% asset tools/vs-code/device_status_bar.png @path %}
 
+[trusted your computer]: /docs/get-started/install/macos#trust
+
```

Comment:
Did you intend to leave this final `<div>` at the bottom of this file, after listing the reference links out?

Reply:
Yes

Reply:
See https://github.com/flutter/website/pull/3823/files for fixes